docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      args:
+        user: opendor-dev
+        uid: 1000
+      context: ./
+      dockerfile: .docker/php/Dockerfile
+    image: php:8.0
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - ./:/var/www
+    depends_on:
+      - db
+    networks:
+      - opendor
+
+  db:
+    image: postgres:12.7
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: ${DB_DATABASE}
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+    volumes:
+      - psql:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    networks:
+      - opendor
+
+  nginx:
+    build:
+      context: ./
+      dockerfile: .docker/nginx/Dockerfile
+    restart: unless-stopped
+    ports:
+      - 8000:80
+      - 80:80
```

Comment:
Could we only keep `8080` host port? Same reason as above with the database.

Reply:
reason was sometimes people type `localhost` and sometimes `localhost:8000` as laravel serve command use this port .. but updating to `8080` only..